ci-operator/step-registry/optional-operators/ci/run-bundle/optional-operators-ci-run-bundle-ref.yaml

```diff
@@ -0,0 +1,35 @@
+ref:
+  as: optional-operators-ci-run-bundle
+  from_image:
+    namespace: ci
+    name: cvp-operator-scorecard
+    tag: v1
+  commands: optional-operators-ci-run-bundle-commands.sh
+  resources:
+    requests:
+      cpu: 300m
+      memory: 300Mi
+  dependencies:
+  - name: bundle-image
+    env: OO_BUNDLE_IMG
+  env:
+  - name: OO_INSTALL_NAMESPACE
+    default: "create"
+    documentation: |
+      The namespace into which the operator and catalog will be installed.
+      Special, default value `create` implies that a new namespace will be created by the step.
+  - name: OO_TARGET_NAMESPACES
+    default: "OwnNamespace"
+    documentation: |
+      The default value `OwnNamespace` means that the Operator will watch its own namespace (from namespace or the kubeconfig default).
+      If the value is `AllNamespaces`, then the Operator will watch all namespaces (cluster-scoped Operators).
+      If the value is `SingleNamespace`, then the Operator will watch a namespace, not necessarily its own.
+      This is an optional parameter, but if the CSV does not support AllNamespaces,
```

Comment:
Looks like this sentence is now outdated - it says `AllNamespaces` but the default is `OwnNamespace`. How about not discussing this in the context of being mandatory/optional parameter, but rather pointing out that it's the users responsibility to set this to a value that their operator actually supports? Related to that: maybe this should not even have a default value and so force the user to intentionally specify something? That would sound better to me if the default value is not guaranteed to always work.

What can the user expect when tthey pass an unsupported value? Does `operator-sdk run bundle` explicitly point that out in a fialure, or will the user observe something like a timeout?